src/Microsoft.Extensions.Configuration.FileExtensions/FileConfigurationProvider.cs

```diff
@@ -104,7 +84,7 @@ private void Load(bool reload)
         /// </summary>
         /// <exception cref="FileNotFoundException">If Optional is <c>false</c> on the source and a
         /// file does not exist at specified Path.</exception>
-        public override void Load()
+        public override void DoLoad()
```

Comment:
BTW, should the version of the method that takes a stream in the file base provider also handle errors?